mobile-browser-based-version/src/components/TaskForm.vue

```diff
@@ -0,0 +1,1051 @@
+<template>
```

Comment:
can you call the file `NewTaskCreationForm`?

Reply:
also we should document somewhere what code people have to change here, in case they add a new feature preprocessing code. how is it linked here, what can be shared and not? 

another question: is there any code-duplication with the UI code of each task? (names, descriptions?) if so, could we only define those in one js place and import here without duplicating them?

Reply:
1. I renamed the required files 
2. Yes I think we should do that. Right now, I believe we have to add another file in `src/task_definition` to add a new preprocessing pipeline.  
3. Actually yes and it's a bit of a problem because it's not limited to the UI code of each task, there is code duplication all over the `vue` files. I think it would be good to create some pre-stylised containers and components that people can reuse